eox_nelp/tests/mixins.py

```diff
@@ -26,6 +27,7 @@ def setUp(self):  # pylint: disable=invalid-name
         self.client = APIClient()
         self.user, _ = User.objects.get_or_create(username="vader")
         self.client.force_authenticate(self.user)
+        setattr(self.user, "profile", Mock(phone_number=None))
```

Comment:
Do you think it could work with that?
```suggestion
        setattr(self.user, "profile", Mock(phone_number=""))
```

Reply:
I was confused. I think that empty in django profile field would be "". But no is None.
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/1f63fed4-2398-4c5a-bbcf-d782dce269f3)